doc/OnlineDocs/contributed_packages/iis.rst

```diff
@@ -1,6 +1,134 @@
+Infeasibility Diagnostics
+!!!!!!!!!!!!!!!!!!!!!!!!!
+
+There are two closely related tools for infeasibility diagnosis:
+
+  - :ref:`Infeasible Irreducible System (IIS) Tool`
+  - :ref:`Minimal Intractable System finder (MIS) Tool`
+
+The first simply provides a conduit for solvers that compute an
+infeasible irreducible system (e.g., Cplex, Gurobi, or Xpress).  The
+second provides similar functionality, but uses the ``mis`` package
+contributed to Pyomo.
+
+
 Infeasible Irreducible System (IIS) Tool
 ========================================
 
 .. automodule:: pyomo.contrib.iis.iis
 
 .. autofunction:: pyomo.contrib.iis.write_iis
+
+Minimal Intractable System finder (MIS) Tool
+============================================
+
+The file ``mis.py`` finds sets of actions that each, independently,
+would result in feasibility.  The zero-tolerance is whatever the
+solver uses, so users may want to post-process output if it is going
+to be used for analysis. It also computes a minimal intractable system
+(which is not guaranteed to be unique).  It was written by Ben Knueven
+as part of the watertap project and is governed by a license shown
```

Comment:
It may be worth putting a link to WaterTAP here so people don't have to guess what it is!